src/tests/eet/eet_test_cache.c

```diff
@@ -60,7 +60,10 @@ EFL_START_TEST(eet_test_cache_concurrency)
    Eina_Bool r;
    int tmpfd;
 
-   file = strdup("/tmp/eet_suite_testXXXXXX");
+   int path_size = strlen(eina_environment_tmp_get()) + strlen("eet_suite_testXXXXXX") + 2;
+   file = malloc(sizeof(char)*path_size);
+   eina_file_path_join(file, path_size
+      , eina_environment_tmp_get(), "eet_suite_testXXXXXX");
```

Comment:
Not a problem but I would suggest it to be more clear:
```suggestion
   const char *filename = "eet_suite_testXXXXXX";
   char *dirname = eina_environment_tmp_get();
   int file_length = strlen(dirname) + strlen(filename) + 2; // <path separator> + <end of string>
	
   file = malloc(sizeof(char) * file_length);
   eina_file_path_join(file, file_length, dirname, filename);
```

Reply:
Not only clear, but less hardcoded strings (which mean more consistency in maintance).

Reply:
Done, please have a look and let me know if there is anything wrong.